SPECS/perl-local-lib/perl-local-lib.spec

```diff
@@ -0,0 +1,320 @@
+Name:           perl-local-lib
+Version:        2.000024
+Release:        9%{?dist}
+Summary:        Create and use a local lib/ for perl modules
+License:        GPL+ OR Artistic
+Group:          Development/Libraries
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Url:            https://metacpan.org/release/local-lib
+Source:         https://cpan.metacpan.org/authors/id/H/HA/HAARG/local-lib-%{version}.tar.gz
+Source10:       perl-homedir.sh
+Source11:       perl-homedir.csh
+BuildArch:      noarch
+# Build
+BuildRequires:  coreutils
+BuildRequires:  make
+BuildRequires:  perl
+BuildRequires:  perl(CPAN)
+BuildRequires:  perl(CPAN::HandleConfig)
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 7.00
+# BuildRequires:  perl(File::HomeDir)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+# Runtime
+BuildRequires:  perl(Carp)
+BuildRequires:  perl(Carp::Heavy)
+BuildRequires:  perl(Config)
+BuildRequires:  perl(File::Basename)
+BuildRequires:  perl(File::Glob)
+# Tests only
```

Comment:
If these are test dependencies, can we gate their inclusion at build-time behind the %with_check variable we set during check builds?

Reply:
added %with_check for tests